### ISSUE TITLE
Compiling errors

### ISSUE THREAD

**Issue by LinkTree3** [xerpi/libds3ps2#1]
Hi, I wanted to ask if you could provide the binary file for this? When I try to build the project using the Makefile I get these errors:

make -C iop   install
make[1]: Entering Directory '/home/username/Downloads/libds3ps2-master/iop'
Makefile:35: /samples/Makefile.pref: No such file or directory
Makefile:36: /samples/Makefile.iopglobal: No such file or directory
make[1]: **\* No rule to make target '/samples/Makefile.iopglobal'.    Stop.
make[1]: Leaving directory '/home/username/Downloads/libds3ps2-master/iop'
make: **\* [all] Error 2

I have very limited experience with to c/c++ and makefiles. I don't know what causes this issues.
I also tried building it in windows 7 using nmake but from what I remember it was saying it doesn't know what "install" is or something like that.

**Comment by xerpi** [comment]
It seems like your SDk isn't installed properly. I'll check my installation and tell you if you are missing some files later today. Also, this is a library for homebrew developers, it doesn't allow you to play commercial games with the DS3.

**Comment by LinkTree3** [comment]
Thank you Sergi! I managed to find a working binary of this and saw it in action. It's pretty cool. I'd still love to know why it doesn't compile for me. Technically speaking, do you think it would be possible for one to implement this library into OPL and make it work with all the games it supports? or will it just allow you to navigate through OPL's menus? I contacted and asked OPL's creator the same question yesterday and am still waiting for his answer But was hoping you might know as well.

**Comment by xerpi** [comment]
Yeah I'd like to mod OPL to make it use a custom pad module that uses my lib so the game will think it's reading from the PS2 controller but it'll actually be the DS3/DS4 one.
I think that we both OPL devs and me should work together in order to achieve that.

**Comment by LinkTree3** [comment]
Have you tried to contact OPL's developer yet? I managed to contact him here with a previous question I had:
ifcaro@ifcaro.net

If you'd like, I'd love to help you with this project if you'll decide to go for it. I have limited experience with C/C++ and PS2 specific programming, but I do have general programming experience and I could help you with testing and maybe graphics.

**Comment by xerpi** [comment]
Yes, I tried contacting ifcaro and SP193 both none of them responded.
About then errors while compiling, can you tell me what you have inside: $(PS2DEV)/ps2sdk/samples ?

**Comment by LinkTree3** [comment]
Ah, I assume you are referring to this library? https://github.com/ps2dev/ps2sdk

I haven't downloaded or installed it. Is it necessary to compile the example? If so how would you go about installing it? Is this an up to date guide or should I do it differently?
https://docs.google.com/document/d/1WEyNu8hu2eKJYODSwzlvsIMxuz7z9jOcYWs3TyJKoGk/edit?pli=1

Thanks!

**Comment by xerpi** [comment]
Yes you need that library. And it seems like it's an updated version so go for it.

**Comment by Sergynx** [comment]
Hey there. Wanted to ask if there was any progress on the lib integration project. I'm quite interested.

**Comment by xerpi** [comment]
Not yet, and I'm so busy and have a lots of projects in mind that I'm not sure if I'll ever be able to work on that.

**Comment by Invictaz** [comment]
Really sad, I hope it comes alive into OPL.

**Comment by haudi** [comment]
#define DS3_VID 0x054C
#define DS3_PID 0x0268

When I change this ID to a PS3 Arcade Sticks one, do I also have to change the complete source code? Sorry, I'm not very good in coding.

**Comment by hellrick63** [comment]
hi guy,
thanks for your job.
have you a solution for an HORI arcade fighting pad?
it works on PS3 and PS4.
thanks
